socorro/unittest/processor/rules/test_breakpad.py

```diff
@@ -464,28 +449,34 @@ def test_success_all_types_of_signatures(self, mocked_subprocess_module):
             "Small | js::irregexp::ExecuteCode<T>",
         ]
         for signature in signatures:
-            processed_crash = DotDict(base_processed_crash)
-            processed_crash.signature = signature
+            processed_crash = copy.copy(base_processed_crash)
```

Comment:
All these copies need to be deepcopies because the rules are changing the processed crash in deep and existentially meaningful ways.